ocs_ci/ocs/scale_lib.py

```diff
@@ -861,7 +865,10 @@ def construct_pvc_creation_yaml_bulk_for_kube_job(no_of_pvc, access_mode, sc_nam
     pvc_dict_list = list()
     for i in range(no_of_pvc):
         pvc_name = helpers.create_unique_resource_name("test", "pvc")
-        size = f"{random.randrange(5, 105, 5)}Gi"
+        if pvc_size is not None:
```

Comment:
I think we can check this condition using `if pvc_size:` if there is some value in param then if part will be executed, and if no data or None then else part will be executed.

Reply:
@keesturam @ramkiperiy Fixed according to your suggestion